.github/workflows/101_build.yml

```diff
@@ -1,14 +1,12 @@
 name: build
 
-permissions: {} 
```

Comment:
Why delete this? It removes default permissions for all jobs that don't explicitly require them, which is a good thing. IMO this should probably prefix every workflow file